util/file.go

```diff
@@ -1,17 +1,17 @@
 package util
 
 import (
-	"os"
-
+	"bytes"
+	"github.com/grtl/mysql-operator/artifacts"
 	"k8s.io/apimachinery/pkg/util/yaml"
 )
 
 // ObjectFromFile creates a kubernetes object from a yaml file.
 func ObjectFromFile(filename string, destination interface{}) error {
-	f, err := os.Open(filename)
+	b, err := artifacts.Asset(filename)
```

Comment:
Let's not use single-character variable names if we can avoid it. Maybe `assetBytes` instead of `b`, `reader` or `assetReader` instead of `r`?